stubs/PyYAML/yaml/__init__.pyi

```diff
@@ -21,20 +23,20 @@ from .tokens import *
 # FIXME: the functions really return str if encoding is None, otherwise bytes. Waiting for python/mypy#5621
 _Yaml: TypeAlias = Any
 
-__with_libyaml__: Any
-__version__: str
-
 _T = TypeVar("_T")
 _Constructor = TypeVar("_Constructor", bound=BaseConstructor)
 _Representer = TypeVar("_Representer", bound=BaseRepresenter)
 
+__with_libyaml__: Any
```

Comment:
https://github.com/yaml/pyyaml/blob/957ae4d495cf8fcb5475c6c2f1bce801096b68a5/lib/yaml/__init__.py#L12-L16

```suggestion
__with_libyaml__: bool
```

Reply:
Sure, may as well while we're here.